workflow-service/pom.xml

```diff
@@ -126,11 +126,6 @@
             <artifactId>workflow-engine</artifactId>
             <version>${revision}</version>
         </dependency>
-        <dependency>
```

Comment:
is there any way to keep it for local profile, and remove it for other profiles, e.g. dev & release ?  so it will be easier for our local debug

Reply:
Yes, we can add new profile [here](https://github.com/redhat-developer/parodos/blob/main/pom.xml#L189)

Reply:
Great! thank you Piotr

Reply:
added